ui/packages/catalog-ui-search/src/main/webapp/styles/mixins.less

```diff
@@ -57,6 +57,9 @@
     opacity: 1;
   }
 }
+.is-split-button button {
```

Comment:
Could you move this above to line 47? `is-split-button button, .is-button { ... }`